statistics/routes/routes.js

```diff
@@ -7,4 +7,5 @@ const router = express.Router();
 router.get('/courses/', courseController.courseList);
 router.get('/courses/:courseCode/', courseController.courseDetail);
 router.get('/results/:courseCode/', courseController.resultList);
+router.get('/latestUpdate/', courseController.latestUpdate);
```

Comment:
Could you add documentation for this route in API.md?

Reply:
Done! See the API documentation update below.